packages/feedback-react/src/followup/useFollowup.ts

```diff
@@ -0,0 +1,103 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+import { FeedbackAnswer, FeedbackOption, FollowupQuestion } from "../types";
+
+export const useFollowup = (questions: FollowupQuestion[], onSubmit: (a: FeedbackAnswer[]) => void) => {
+    const [values, setValues] = useState<FeedbackAnswer[]>();
+    const [step, setStep] = useState({
```

Comment:
Å lage states i form av objekter istedenfor primitive verdier kan potensielt føre til unødvendige re-renders fordi det er vanskeligere å forstå når staten har endret seg

Reply:
Det er bare et issue dersom man oppdaterer enkeltverdier inne i objektet. Her vil _hele objektet_ endre seg for hvert steg, så det vil ikke være noe problem 😊 Jeg kan sikkert gjøre det litt mer eksplisitt tydelig her ved å sette den initielle verdien som en navngitt konstant.